src/Repository/IdeaRepository.php

```diff
@@ -126,15 +126,20 @@ public function getSupportersSlackId(int $ideaId): array
             ->getResult();
     }
 
-    public function getSupportersSortIdea(): array
+    public function getSupportersSortIdea(array $criteria = []): array
```

Comment:
Why do you pass an criteria array when a simple bool could be enough

Reply:
who can do more can do less :D